s4e-web/src/app/views/map-view/state/overlay/overlay.factory.spec.ts

```diff
@@ -4,5 +4,6 @@ import {Overlay} from './overlay.model';
 export const OverlayFactory = Factory.makeFactory<Overlay>({
   id: Factory.each(i => `overlay:${i}`),
   caption: Factory.each(i => `Overlay #${i}`),
-  type: 'wms'
+  type: 'wms',
+  favourite: Factory.each(i => !!Math.round(Math.random()))
```

Comment:
This is generally not a good idea, as it's rather useless in tests (non deterministic input). In most cases if we do not expect fields to be unique settling for static value is a better idea. Eg. `false` in this case.

Reply:
ok